Source/WebKit/UIProcess/API/APIPageConfiguration.h

```diff
@@ -155,140 +155,140 @@ class PageConfiguration : public ObjectImpl<Object::Type::PageConfiguration> {
+        std::optional<bool> delaysWebProcessLaunchUntilFirstLoad;
+        std::optional<double> cpuLimit;
 
-    WTF::String m_overrideContentSecurityPolicy;
+        WTF::String overrideContentSecurityPolicy;
 
 #if PLATFORM(COCOA)
-    WTF::Vector<WTF::String> m_additionalSupportedImageTypes;
+        WTF::Vector<WTF::String> additionalSupportedImageTypes;
 #endif
 
 #if ENABLE(APPLICATION_MANIFEST)
-    RefPtr<ApplicationManifest> m_applicationManifest;
+        RefPtr<ApplicationManifest> applicationManifest;
 #endif
 
-    HashMap<WTF::String, Ref<WebKit::WebURLSchemeHandler>> m_urlSchemeHandlers;
-    Vector<WTF::String> m_corsDisablingPatterns;
-    HashSet<WTF::String> m_maskedURLSchemes;
-    bool m_userScriptsShouldWaitUntilNotification { true };
-    bool m_crossOriginAccessControlCheckEnabled { true };
-    WTF::String m_processDisplayName;
-    bool m_loadsSubresources { true };
-    std::optional<MemoryCompactLookupOnlyRobinHoodHashSet<WTF::String>> m_allowedNetworkHosts;
-    
+        HashMap<WTF::String, Ref<WebKit::WebURLSchemeHandler>> urlSchemeHandlers;
+        Vector<WTF::String> corsDisablingPatterns;
+        HashSet<WTF::String> maskedURLSchemes;
+        bool userScriptsShouldWaitUntilNotification { true };
+        bool crossOriginAccessControlCheckEnabled { true };
+        WTF::String processDisplayName;
+        bool loadsSubresources { true };
+        std::optional<MemoryCompactLookupOnlyRobinHoodHashSet<WTF::String>> allowedNetworkHosts;
+
 #if ENABLE(APP_BOUND_DOMAINS)
-    bool m_ignoresAppBoundDomains { false };
-    bool m_limitsNavigationsToAppBoundDomains { false };
+        bool ignoresAppBoundDomains { false };
+        bool limitsNavigationsToAppBoundDomains { false };
 #endif
 
-    bool m_mediaCaptureEnabled { false };
-    bool m_httpsUpgradeEnabled { true };
+        bool mediaCaptureEnabled { false };
+        bool httpsUpgradeEnabled { true };
 
-    WebCore::ShouldRelaxThirdPartyCookieBlocking m_shouldRelaxThirdPartyCookieBlocking { WebCore::ShouldRelaxThirdPartyCookieBlocking::No };
-    WTF::String m_attributedBundleIdentifier;
-
-#if PLATFORM(IOS_FAMILY)
-    WebKit::AttributionOverrideTesting m_appInitiatedOverrideValueForTesting { WebKit::AttributionOverrideTesting::NoOverride };
-#endif
+        WebCore::ShouldRelaxThirdPartyCookieBlocking shouldRelaxThirdPartyCookieBlocking { WebCore::ShouldRelaxThirdPartyCookieBlocking::No };
+        WTF::String attributedBundleIdentifier;
 
 #if HAVE(TOUCH_BAR)
-    bool m_requiresUserActionForEditingControlsManager { false };
+        bool requiresUserActionForEditingControlsManager { false };
 #endif
 
-    WebCore::ContentSecurityPolicyModeForExtension m_contentSecurityPolicyModeForExtension { WebCore::ContentSecurityPolicyModeForExtension::None };
+        WebCore::ContentSecurityPolicyModeForExtension contentSecurityPolicyModeForExtension { WebCore::ContentSecurityPolicyModeForExtension::None };
+    };
+
+    Data m_data;
```

Comment:
Could you add a comment that new member should be added to `Data` to avoid breaking copy?